src/wwwroot/controllers/MainCtrl.js

```diff
@@ -14,11 +14,16 @@
     '$log',
     'ModalService',
     '$route',
-    '$interval'
+    '$interval',
+    '$translate',
+    'utils'
   ];
 
-  function MainCtrl($rootScope, $scope, $window, $location, auth, $log, ModalService, $route, $interval) {
-    
+  function MainCtrl($rootScope, $scope, $window, $location, auth, $log, ModalService, $route, $interval, $translate, utils) {
+     
+    var key = utils.getPreferredLanguage();
+    $translate.use(key);
```

Comment:
What happen when _key_ is null? should we deal with that scenario?

Reply:
If key is null is just ignored and use English as default language.